OpenDreamServer/Dream/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -79,6 +80,17 @@ static class DreamProcNativeRoot {
 
             return new DreamValue(Convert.ToChar(ascii).ToString());
         }
+        
+        [DreamProc("clamp")]
+        [DreamProcParameter("Value", Type = DreamValueType.Number)]
+        [DreamProcParameter("Low", Type = DreamValueType.Number)]
+        [DreamProcParameter("High", Type = DreamValueType.Number)]
+        public static DreamValue NativeProc_ckey(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
```

Comment:
`clamp()` can also take a list as its first argument. It needs to handle this.

Also, move this below ckey.